### ISSUE TITLE
[Serious] Text content lacks 4.5:1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#597]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87708 |
| **Checkpoint** | 1.4 Distinguishable › Color Contrast (regular text) (1.4.3.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 02 Import/Export Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/11/26 at 9:41 am |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The contrast ratio between text and its background is not at least 4.5:1.
On Export Settings, select a few checkboxes, then Export.  In the resulting dialog, there is a blue link on pink background in the Warning section.
Also occurs on Workflows, click a workflow to view then select Export, that dialog box has a Warning section.

## Recommendation

Rule : 
Small text (under 18 point regular font or 14 point bold font) MUST have a contrast ratio of at least 4.5 to 1 with the background.

How To Fix: 
Fix this issue by adjusting the text and/or background to increase the contrast to at least 4.5 to 1.

Background: 
People who have low vision or are colorblind may have difficulty reading text if the contrast between the text its background is insufficient. When the contrast ratio between text and its background is adequate, people who have low vision or are colorblind are more likely to be able to read the text.

## Screenshots

![02_blue pink contrast](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87708/02_blue_pink_contrast.png)
![05_blue pink contrast](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87708/05_blue_pink_contrast.png)
